### PR TITLE
Added EMAIL_SUBJECT_PREFIX to settings.py

### PR DESCRIPTION
**Pull request by anantakash73**
Set EMAIL_SUBJECT_PREFIX to 'Student Explorer'

Possible issue
Ideally this should be specified in the .env file but I did not want to make any change to that without authorization

Secondly, I will be working on moving to django-constance after this

Thirdly, I do not know how to test this change as I don't think I can send emails

### CODE REVIEW

File: student_explorer/settings.py [hunk 1]
```diff
@@ -158,6 +158,8 @@
 
 FEEDBACK_EMAIL = config('DJANGO_FEEDBACK_EMAIL', default=None)
 
+EMAIL_SUBJECT_PREFIX = 'Student Explorer'
```
Comment (jonespm) [inline student_explorer/settings.py]: This should be configurable the same way. Can you make this change and also update the sample .env.sample file documenting this? Just using EMAIL_SUBJECT_PREFIX seems fine to me.

```suggestion
EMAIL_SUBJECT_PREFIX = config('EMAIL_SUBJECT_PREFIX', 'Student Explorer')
```

**Comment by jonespm** [comment]
Thanks, sorry for the delay on the review.

If you don't get Django constance implemented that's fine too, not sure how much effort you needed to put into this. I gave some feedback.

It looks like you commented a little about the testing of this on the issue, I'll comment there.

**Comment by anantakash73** [comment]
Just made those changes

File: student_explorer/settings.py [hunk 2]
```diff
@@ -156,7 +156,10 @@
 EMAIL_USE_TLS = config('DJANGO_EMAIL_USE_TLS', default=None)
 EMAIL_USE_SSL = config('DJANGO_EMAIL_USE_SSL', default=None)
 
-FEEDBACK_EMAIL = config('DJANGO_FEEDBACK_EMAIL', default=None)
+FEEDBACK_EMAIL = config('DJANGO_FEEDBACK_EMAIL', default='aanant@umich.edu')
```
Comment (jonespm) [inline student_explorer/settings.py]: Just leave this one default none
```suggestion
FEEDBACK_EMAIL = config('DJANGO_FEEDBACK_EMAIL', default=None)
```

Comment (jonespm) [reply student_explorer/settings.py]: You should be able to test this by changing the value in the .env rather than changing it here.

**Comment by anantakash73** [comment]
Accepted @jonespm suggestion

**Pull request merged**